pkgs/dart_pad/lib/embed.dart

```diff
@@ -306,19 +306,19 @@ class Embed extends EditorUi {
       DElement(querySelector('#issues-toggle')!),
       snackbar,
     )..onItemClicked.listen((item) {
-        if (item.sourceName == 'test.dart') {
-          // must be test editor
-          if (!_showTestCode) {
-            _showTestCode = true;
-            showTestCodeCheckmark.toggleClass('hide', !_showTestCode);
-            tabController.setTabVisibility('test', _showTestCode);
-          }
-          tabController.selectTab('test');
-          _jumpToTest(item.line, item.charStart, item.charLength, focus: true);
-        } else {
-          tabController.selectTab('dart');
-          _jumpTo(item.line, item.charStart, item.charLength, focus: true);
-        }
+        // if (item.sourceName == 'test.dart') {
+        //   // must be test editor
+        //   if (!_showTestCode) {
+        //     _showTestCode = true;
+        //     showTestCodeCheckmark.toggleClass('hide', !_showTestCode);
+        //     tabController.setTabVisibility('test', _showTestCode);
+        //   }
+        //   tabController.selectTab('test');
+        //   _jumpToTest(item.line, item.charStart, item.charLength, focus: true);
```

Comment:
This is required for codelabs like the [layout basics](https://docs.flutter.dev/codelabs/layout-basics) codelab. Is there a reason this is commented out?

Reply:
I added this back in. When updating this frontend, I'd assumed we didn't actually have more then one file. Re-reading this, it looks like we have two tabs (src and tests), which are analyzed as one file, but then any analysis issues reassigned back into two files after.

I added this back.